src/main/java/com/openshift/jenkins/plugins/pipeline/Auth.java

```diff
@@ -11,38 +11,47 @@
 import java.io.UnsupportedEncodingException;
 import java.net.URI;
 import java.security.KeyStore;
+import java.security.cert.Certificate;
 import java.security.cert.CertificateFactory;
+import java.security.cert.CertificateParsingException;
 import java.security.cert.X509Certificate;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
+import javax.net.ssl.SSLPeerUnverifiedException;
 import javax.net.ssl.SSLSession;
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 import javax.net.ssl.X509TrustManager;
 
 import com.openshift.restclient.ISSLCertificateCallback;
 
-import hudson.EnvVars;
-import hudson.model.AbstractBuild;
-import hudson.model.Run;
 import hudson.model.TaskListener;
 
 
 public class Auth implements ISSLCertificateCallback {
+    static final Logger LOGGER = Logger.getLogger(Auth.class.getName());
+    
 	private static final String AUTH_FILE = "/run/secrets/kubernetes.io/serviceaccount/token";
 	private static final String CERT_FILE = "/run/secrets/kubernetes.io/serviceaccount/ca.crt";
 	public static final String CERT_ARG = " --certificate-authority=/run/secrets/kubernetes.io/serviceaccount/ca.crt ";
+	public static final String SSL_PROXY_HOSTS = "SSL_PROXY_HOSTS";
```

Comment:
No, this is just a backdoor config to provide a specific list of hosts to verify against if the solution overlooked some valid scenario.